### PR TITLE
doc: Add comment in contributing file indicating that Node v12.19 is …

### PR DESCRIPTION
**Pull request by Moneyba**
…required

## Description
Add a comment in the contributing file indicating that Node v12.19 is required.

## Changes
<Changes made> 

## Checks
- [ ] Project Builds
- [ ] Project passes tests and checks
- [ ] Updated documentation accordingly

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -141,6 +141,7 @@ To start contributing to the project you would need to set up the project in you
   - `./scripts/check-all-the-things.sh` on Linux and MacOS
   - `.\scripts\check-all-the-things.ps1` on Windows
 
+**Important:** Node v12.19 is required.
```
Comment (rdiaz82) [inline CONTRIBUTING.md:144]: I would say that this is the minimim required version, because it sounds that we are supporting just this version

**Review by rdiaz82 — CHANGES_REQUESTED** [review]
I left a nit-picky comment

File: CONTRIBUTING.md (hunk 1 above)
Comment (javiertoledo) [reply CONTRIBUTING.md:144]: Single-click suggestion:
```suggestion
**Important:** Node version v12.19 or greater is required.
```

Comment (javiertoledo) [reply CONTRIBUTING.md:144]: It looks like that's actually the case :trollface: https://github.com/boostercloud/booster/issues/494

**Review by javiertoledo — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (alvaroloes) [reply CONTRIBUTING.md:144]: I have node v14 and everything runs well.

In any case, I would add the "minimum" part and then try to fix the problems with node 15

Comment (javiertoledo) [reply CONTRIBUTING.md:144]: Yeah, it's weird that something works only under a specific version, ideally it should work starting in some specific version and in all the next...

**Comment by Moneyba** [comment]
I finally made the suggested change :)

**Comment by javiertoledo** [comment]
Alright! I'll merge this right away, thanks @Moneyba!

**Pull request merged**